edi_oca/models/edi_exchange_consumer_mixin.py

```diff
@@ -24,21 +25,19 @@ class EDIExchangeConsumerMixin(models.AbstractModel):
         domain=lambda r: [("model", "=", r._name)],
     )
     exchange_record_count = fields.Integer(compute="_compute_exchange_record_count")
-    expected_edi_configuration = Serialized(
-        compute="_compute_expected_edi_configuration",
+    edi_form_config = Serialized(
```

Comment:
this can be probably made even more generic and named `edi_config` while keeping `edi_has_form_config` which is specific.

Reply:
Ok, I changed this to be more generic so that we can reuse it for any kind of lookup of configuration on consumer models.